docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,241 @@
+| .README.md.txt | Main README file for GAMMA InSAR | S1AB_20171111T150004_20171117T145926_VVP006_INT80_G_ueF_4D09.README.md.txt |
+
+*Table 3: Metadata files in product package*
+
+### Options 
+
+There are several options offered with the InSAR products.  Currently, these are (1) the number of looks to take, (2) inclusion of look vectors, and (3) inclusion of the line of sight displacement file.
+
+The number of looks drives the pixel spacing of the output products.  Selecting 10x2 looks will yield larger products with 80 m pixel spacings.  Selecting 20x4 looks reduces the resolution to 160 m and reduces the size of the products (roughly 1/4 the size of 10x2 look products).  The default is 20x4 looks.
+
+The look vectors are stored in two files.  The lv_theta indicates the SAR look vector elevation angle at each pixel, ranging from -pi/2 (down) to pi/2 (up). The look vector elevation angle is defined as the angle between the horizontal surface and the look vector with positive angles indicating sensor positions above the surface. The lv_phi map indicates the SAR look vector orientation angle at each pixel, ranging from 0 (east) to pi/2 (north). The look vector orientation angle is defined as the angle between the East direction and the projection of the look vector on the horizontal surface plan. The orientation angle increases towards north, with the North direction corresponding to &#960/2 (and south to -&#960/2). Both angles are expressed in radians.  The default is to not include these files in the output product bundle.
+
+The line-of-sight displacement is the ground movement away from or towards the platform.  It is used to create the vertical displacement map during the final steps of InSAR processing. In order to have this file included in the output zip file, this option must be selected.  The default is to not include the line-of-sight data file.
+
+## Limitations
+
+The baseline is defined as the difference of the platform positions when a given area is imaged.  HyP3 baselines are calculated using the best state vectors available.  These are either restituted or precise.  **If no restituted or precise state vectors are available, the process will not run**.
+
+## Error Sources
+
+### Atmospheric Delay
+While SAR signals can penetrate clouds, atmospheric conditions can delay the transmission of the signal. This results in phase differences that can look like surface deformation signals, but are actually driven by differences in the atmospheric conditions between the pair of acquisitions used to generate the interferogram. 
+
+In some cases, atmospheric errors can be corrected by using an atmospheric model to remove the impacts of the turbulent delay from the interferogram. Another approach is to use time series analysis to identify outliers.
+
+ALWAYS DOUBT YOUR INTERFEROGRAM FIRST! View the interferogram critically, and consider if fringe patterns could potentially be driven by atmospheric effects. 
```

Comment:
To a casual reader, this seems a bit alarmist?  Would `Always view your interferogram critically, and consider...` be sufficient?

Reply:
I think it *is* alarming. I guess we could tone it down some, but it's definitely important for people to look critically at their interferogram. There have been false alarms (resulting in evacuations, etc.) when InSAR seemed to be indicating volcanic activity that was really just atmospheric impacts. Users who are new to interferograms could read things in that are not there, and it can be difficult to tell for certain even for more experienced users. Stressing that a single interferogram should not be trusted is key.

Reply:
Addressed in PR #96